spec/dummy/config/environments/production.rb

```diff
@@ -20,7 +20,7 @@
   # config.action_dispatch.rack_cache = true
 
   # Disable Rails's static asset server (Apache or nginx will already do this).
-  config.serve_static_assets = false
+  config.serve_static_files = false
```

Comment:
@rossmeissl I don't think this change is backwards-compatible with Rails 4.1. This change is only for the spec/dummy. I can be easily ignore, the spec/dummy can stay in Rails 4.1 instead of 4.2 (I've tested with both), up to you.